Wikipedia/Code/WMFTableOfContentsPresentationController.swift

```diff
@@ -55,11 +60,21 @@ public class WMFTableOfContentsPresentationController: UIPresentationController
         self.containerView!.addSubview(self.backgroundView)
         self.containerView!.addSubview(self.presentedView()!)
         
+        // Add a status bar background so the TOC scroll below it
+        self.statusBarBg = UIView(frame: CGRectMake(self.containerView!.frame.origin.x, 0, self.containerView!.frame.size.width, UIApplication.sharedApplication().statusBarFrame.size.height))
```

Comment:
use `CGRect()` initializer instead of `CGRectMake`. checkout [this blog post](https://medium.com/swift-programming/swift-cgrect-cgsize-cgpoint-5f4196da9cf8?utm_campaign=This%2BWeek%2Bin%2BSwift&utm_medium=web&utm_source=This_Week_in_Swift_73#.wc1vodie0) for pointers on how to CGGeometry in swift